README.md

```diff
@@ -41,6 +41,7 @@ The data contained in a `Snapshot` per particle is:
 - `typeid`: (*N*,) array of type indexes (dtype: `int`, default: `1`)
 - `mass`: (*N*,) array of masses (dtype: `float`, default: `1`)
 - `charge`: (*N*,) array of charges (dtype: `float`, default: `0`)
+- `type_label`: Labels of particle typeids. (dtype: `LabelMap`, default: `None`)
```

Comment:
Since this is not a NumPy array or per-particle, it should be moved to a separate section. The `bonds`, etc. should actually also do the same.
```suggestion

The optional topology data is:

- `type_label`: Labels of particle typeids. (`LabelMap`, default: `None`)
```
I realized that the ones below also have the same issue.